labs/firepower-restapi-105/1.md

```diff
@@ -1,38 +1,29 @@
 # FMC - REST - API - 105
 
-## Lab Objective
+## Lab objective
 
-Basic understanding of debugging the FMC REST API errors
+This lab gives a basic understanding of debugging the FMC REST API errors.
 
-Completion time: 10 minutes
 
 ## Prerequisites
-Basic understanding of REST principles
+It's best if you have a basic understanding of REST principles.
 
 
-## Learning Objective
-After Completing this module you will be able to:
-Get basic understanding of debugging FMC REST APIs.
+## Learning objective
```

Comment:
I think this entire section under "Learning Objective" can be deleted since it's repeating the "Lab" Objectives above.